libcxx/include/__tree

```diff
@@ -1025,6 +1025,58 @@ public:
   _LIBCPP_HIDE_FROM_ABI iterator __node_insert_multi(__node_pointer __nd);
```

Comment:
Can you update the tests for these operations like you did for the multi versions?